samples/matter/lock/README.rst

```diff
@@ -560,119 +560,40 @@ Upgrading the device firmware
 
 To upgrade the device firmware, complete the steps listed for the selected method in the :doc:`matter:nrfconnect_examples_software_update` tutorial of the Matter documentation.
 
-.. _matter_lock_enabling_scheduled_timed_access:
-
-Enabling scheduled timed access
-===============================
-
-To enable the scheduled timed access feature, complete the following steps:
-
-1. Enable all needed scheduled timed access types in the ZAP file:
-
-   a. Open the :file:`lock.zap` file using the following west command:
-
-      .. code-block:: console
-
-         west zap-gui
-
-   #. Select the endpoint that contains the Matter Door Lock cluster.
-      By default, this is `Endpoint-1`.
-
-   #. Click the :guilabel:`Configure` symbol for the ``Door Lock`` cluster entry.
-   #. In the **Door Lock** context window, go to the **Attributes** tab and enable all required attributes:
-
-      * ``NumberOfWeekDaySchedulesSupportedPerUser`` for the ``Week-day`` schedule support.
-      * ``NumberOfYearDaySchedulesSupportedPerUser`` for the ``Year-day`` schedule support.
-      * ``NumberOfHolidaySchedulesSupported`` for the ``Holiday`` schedule support.
-
-   #. In the **Door Lock** context window, go to the **Attribute Reporting** page and enable all required attribute reporting entries:
-
-      * ``NumberOfWeekDaySchedulesSupportedPerUser`` for the ``Week-day`` schedule support.
-      * ``NumberOfYearDaySchedulesSupportedPerUser`` for the ``Year-day`` schedule support.
-      * ``NumberOfHolidaySchedulesSupported`` for the ``Holiday`` schedule support.
-
-   #. In the **Door Lock** context window, go to the **Commands** page and enable all required command entries:
-
-      * For the ``Week-day`` schedule support:
+.. _matter_lock_snippets:
 
-        * ``SetWeekDaySchedule``
-        * ``GetWeekDaySchedule``
-        * ``GetWeekDayScheduleResponse``
-        * ``ClearWeekDaySchedule``
+Snippets
```

Comment:
Snippets section should be under Configuration, after the custom configurations.